### PR TITLE
[iOS] Update swap confirmation

### PR DESCRIPTION
**Pull request by thangpham26394**
RE #KRYS-2023

### CODE REVIEW

File: KyberNetwork/KyberNetwork/Screens/CommonViews/Confirms/SwapConfirm/ViewModel/KConfirmSwapViewModel.swift [hunk 1]
```diff
@@ -28,12 +28,24 @@ struct KConfirmSwapViewModel {
     let usd = self.transaction.expectedReceive * BigInt(rate.usd * pow(10.0, 18.0)) / BigInt(10).power(self.transaction.to.decimals)
     return usd
   }
+  
+  var fromUSDAmount: BigInt? {
+    guard let rate = KNTrackerRateStorage.shared.getPriceWithAddress(self.transaction.from.address) else { return nil }
+    let usd = self.transaction.amount * BigInt(rate.usd * pow(10.0, 18.0)) / BigInt(10).power(self.transaction.to.decimals)
```
Comment (toidv) [inline KyberNetwork/KyberNetwork/Screens/CommonViews/Confirms/SwapConfirm/ViewModel/KConfirmSwapViewModel.swift:34]: Wrong decimal usage, it must be from decimal

File: KyberNetwork/KyberNetwork/Screens/CommonViews/Confirms/SwapConfirm/ViewModel/KConfirmSwapViewModel.swift [hunk 2]
```diff
@@ -28,12 +28,24 @@ struct KConfirmSwapViewModel {
...
 
   var displayEquivalentUSDAmount: String? {
     guard let amount = self.equivalentUSDAmount, !amount.isZero else { return nil }
     let value = amount.displayRate(decimals: 18)
     return "~ $\(value) USD"
   }
+  
+  var displayFromUSDAmount: String? {
+    guard let amount = self.fromUSDAmount, !amount.isZero else { return nil }
+    let value = amount.displayRate(decimals: 18)
```
Comment (toidv) [inline KyberNetwork/KyberNetwork/Screens/CommonViews/Confirms/SwapConfirm/ViewModel/KConfirmSwapViewModel.swift:46]: Must use USD type decimal (up to 2 decimals)

Comment (toidv) [reply KyberNetwork/KyberNetwork/Screens/CommonViews/Confirms/SwapConfirm/ViewModel/KConfirmSwapViewModel.swift:46]: Also if `value` (after rounding) is zero please leave as empty ""

File: KyberNetwork/KyberNetwork/Screens/CommonViews/Confirms/SwapConfirm/ViewModel/KConfirmSwapViewModel.swift [hunk 3]
```diff
@@ -29,9 +29,27 @@ struct KConfirmSwapViewModel {
...
     guard let amount = self.equivalentUSDAmount, !amount.isZero else { return nil }
-    let value = amount.displayRate(decimals: 18)
+    let value = amount.string(decimals: 18, minFractionDigits: 0, maxFractionDigits: DecimalNumber.usd)
+    if let doubleValue = Double(value), doubleValue < 0.01 {
+      return ""
+    }
+    return "~ $\(value) USD"
+  }
+  
+  var displayFromUSDAmount: String? {
```
Comment (toidv) [inline KyberNetwork/KyberNetwork/Screens/CommonViews/Confirms/SwapConfirm/ViewModel/KConfirmSwapViewModel.swift:47]: Please also format display usd value for transaction fee

**Review by MrQKyber — APPROVED** [review]
LGTM

**Pull request merged**